src/components/input/InputDecorator.tsx

```diff
@@ -73,7 +74,8 @@ const InputDecorator: React.FC<InputDecoratorProps> = ({
     () =>
       child
         ? React.cloneElement(child, {
-          ...rest,
```

Comment:
rest is not useful anymore?

Reply:
It was a mistake. 'rest' is the args of the InputDecorator.